api/DragEvent.json

```diff
@@ -47,6 +47,55 @@
           "deprecated": false
         }
       },
+      "DragEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DragEvent/DragEvent",
+          "description": "<code>DragEvent</code> constructor",
```

Comment:
Shouldn't this be `<code>DragEvent()</code> constructor`?

Reply:
I wasn't aware, but you're right, I fixed it in aa4a4e0 and will submit a PR to fix other occurrences of this mistake.

Reply:
See #2458.